qubes/ext/audio.py

```diff
@@ -20,69 +20,96 @@
 import qubes.config
 import qubes.ext
 
+
 class AUDIO(qubes.ext.Extension):
     # pylint: disable=unused-argument
     @staticmethod
     def attached_vms(vm):
         for domain in vm.app.domains:
-            if getattr(domain, 'audiovm', None) and domain.audiovm == vm:
+            if getattr(domain, "audiovm", None) and domain.audiovm == vm:
```

Comment:
```suggestion
            if getattr(domain, "audiovm", None) == vm:
```

should be equivalent